vmdb/spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -22,17 +22,20 @@
       assert_specific_container_node
       assert_specific_container_service
       assert_specific_container_replicator
+      assert_specific_container_image_and_registry
     end
   end
 
   def assert_table_counts
-    ContainerGroup.count.should        == 2
-    ContainerNode.count.should         == 1
-    Container.count.should             == 3
-    ContainerService.count.should      == 6
-    ContainerPortConfig.count.should   == 2
-    ContainerDefinition.count.should   == 3
-    ContainerReplicator.count.should   == 2
+    ContainerGroup.count.should == 2
+    ContainerNode.count.should == 1
+    Container.count.should == 3
+    ContainerService.count.should == 6
+    ContainerPortConfig.count.should == 2
+    ContainerDefinition.count.should == 3
+    ContainerReplicator.count.should == 2
```

Comment:
> To be strict the ones above you shouldn't touch (as they're not part of this PR)

I had to add spaces because ```ContainerImageRegistry``` is the longest. Reverting.